src/pages/academy/grading/subcomponents/GradingEditor.tsx

```diff
@@ -236,7 +236,20 @@ const GradingEditor: React.FC<GradingEditorProps> = props => {
 
       <div className="grading-editor-header">
         <H3>
-          Currently Grading: {props.studentName} ({props.studentUsername})
+          Currently Grading:
+          <br />
+          {props.studentName
+            ? props.studentName.split(',').map((student, index) => {
+                const usernameList = props.studentUsername ? props.studentUsername.split(',') : [];
+                const username = usernameList.length > index ? usernameList[index].trim() : '';
+                return (
+                  <div key={index}>
+                    {student.trim()} ({username && <span>{username}</span>})
+                    <br />
+                  </div>
+                );
+              })
+            : props.studentName}
```

Comment:
The ternary expression seems unnecessary, since the studentName will always be defined.

Also, source of bug. Lots of Singapore names have a comma (e.g. "Tan Jia Jun, John"). If you want to support these kinds of scenario, you should just pass the array directly to the component, i.e. `studentName` and `studentUsername` should be `studentNames` and `studentUsernames` and each have type `string[]`.

Reply:
Resolved.